x-pack/plugins/security_solution/public/detections/components/rules/severity_badge/index.tsx

```diff
@@ -10,19 +10,21 @@ import React from 'react';
 
 import { euiLightVars } from '@kbn/ui-shared-deps-src/theme';
 import { HealthTruncateText } from '../../../../common/components/health_truncate_text';
+
+const { euiColorVis0, euiColorVis5, euiColorVis7, euiColorVis9 } = euiLightVars;
+const severityToColorMap: Record<string, string> = {
+  low: euiColorVis0,
+  medium: euiColorVis5,
+  height: euiColorVis7,
+};
```

Comment:
Looks like a typo - should be `high` instead of `height`.

I'd suggest to make it type safe and use `Severity` union type from `packages/kbn-securitysolution-io-ts-alerting-types/src/severity/index.ts`. Let's have all 4 severity values in the map.

Reply:
thank you for catching this, I've refactored it to use proposed type